From 967340cd26e5f8022211808f07cc4c3a38219fd1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sat, 2 Nov 2024 17:15:25 +0100
Subject: [PATCH] Queue draw after setting paintable

Without this, the animation is not rendered on the screen.
---
 cartridges/game_cover.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cartridges/game_cover.py b/cartridges/game_cover.py
index 46dd6740c..e6518f947 100644
--- a/cartridges/game_cover.py
+++ b/cartridges/game_cover.py
@@ -117,6 +117,7 @@ def set_texture(self, texture: Gdk.Texture) -> None:
         else:
             for picture in self.pictures:
                 picture.set_paintable(texture or self.placeholder)
+                picture.queue_draw()
 
     def update_animation(self, data: GdkPixbuf.PixbufAnimation) -> None:
         if self.animation == data[1]:
